extension/manifest.json

```diff
@@ -39,7 +39,7 @@
       "matches": [
           "<all_urls>"
       ],
-      "js": ["js/add_input_icon.js"]
+      "js": ["js/add_input_icon.js", "js/fill_relay_address.js"]
```

Comment:
Is it better (performance-wise?) for add-ons to inject multiple content scripts like this, or combine functionality into a single content script like `add_input_icon_and_fill_addresses.js` ?

Reply:
I am not sure. I created a new file because the existing content script has a functionality-specific filename, and I didn't want to put something unrelated into that. We could merge the two together and rename the file to something more generic.

@rpl, you know know which is more performant?